otp/imx6ull-otp/imx6ull-otp.c

```diff
@@ -567,36 +588,34 @@ int main(int argc, char **argv)
 			common.get_uid = 1;
 			break;
 		case 'm':
-			if (argc < 5) {
-				common.display_usage = 1;
-				break;
+			if (argc - (optind - 1) < 3) {
+				usage(argv[0]);
+				return 1;
```

Comment:
`EXIT_FAILURE`/`EXIT_SUCCESS`? Applies to other cases

Reply:
thx, done.